react-notification-system-redux/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```

Comment:
Please add `"noImplicitThis": true,` here. (We've fixed the new-package script in the meanwhile.)